src/cdk-experimental/ui-patterns/behaviors/list-typeahead/list-typeahead.ts

```diff
@@ -37,7 +37,7 @@ export class ListTypeahead<T extends ListTypeaheadItem> {
   navigation: ListNavigation<T>;
 
   /** Keeps track of the characters that typeahead search is being called with. */
-  private _query = signal('');
+  query = signal('');
```

Comment:
Is it better if we leave this private? i.e. should there instead be a public computed for whether a query is present?